Controller.php

```diff
@@ -280,4 +281,23 @@ private function getPrettyDateForSite($datetime, $period, $timezone)
 
         return $prettyDate;
     }
+
+    private function fetchSites($alert)
+    {
+        $sites = SitesManagerApi::getInstance()->getSitesWithAtLeastViewAccess();
```

Comment:
I was considering such case. Isn't original site selector also problematic? When you have 50k sites, all of them would be placed in select widget which also wouldn's be usable. Maybe some hard limit up to how many websites selecting multiple websites is available would be a solution? Selecting many websites in custom alerts is needed for users/clients who want to have alerts for all/most of their websites. I guess in case of 50k websites user is not interested in receiving alerts for each of them (to much emails).